crates/red_knot_python_semantic/resources/mdtest/snapshots/invalid_argument_type.md_-_Invalid_argument_type_diagnostics_-_Many_parameters.snap

```diff
@@ -21,19 +21,19 @@ mdtest path: crates/red_knot_python_semantic/resources/mdtest/diagnostics/invali
 # Diagnostics
 
 ```
-error: lint:invalid-argument-type
+error: lint:invalid-argument-type: argument to this function is incorrect
  --> /src/mdtest_snippet.py:4:8
   |
 2 |     return x * y * z
 3 |
 4 | foo(1, "hello", 3)  # error: [invalid-argument-type]
-  |        ^^^^^^^ Object of type `Literal["hello"]` cannot be assigned to parameter 2 (`y`) of function `foo`; expected type `int`
+  |        ^^^^^^^ expected `int`, found `Literal["hello"]`
   |
-info
- --> /src/mdtest_snippet.py:1:17
+info: function defined here
+ --> /src/mdtest_snippet.py:1:5
   |
 1 | def foo(x: int, y: int, z: int) -> int:
-  |                 ------ parameter declared in function definition here
+  |     ^^^         ------
```

Comment:
This seems to be a slightly unfortunate regression? Previously, we would mark the precise parameter in the "info" span, but now we only highlight the function name. Is this intentional/temporary?

Reply:
It's still there! I just removed the message part. But there is `^^^` for the function name and `------` for the parameter. Maybe I went a little too spartan and I should add the message back saying, "parameter declared here"?

Reply:
Oh, haha :see_no_evil:. I think I completely ignored it because that new `^^^` span was diff-highlighted in dark green, whereas the `------` span just stayed as it was.

In this case, feel free to ignore my comment, but I *think* I would prefer a short "parameter declared here", yes.

Reply:
I think I agree. I've added a message back!